src/node/internal/internal_fs_callback.ts

```diff
@@ -534,16 +535,34 @@ export function mkdir(
 
 export function mkdtemp(
   prefix: FilePath,
-  optionsOrCallback: SingleArgCallback<string> | MkdirTempSyncOptions,
+  optionsOrCallback:
+    | SingleArgCallback<string>
+    | MkdirTempSyncOptions
+    | BufferEncoding
+    | 'buffer'
+    | null,
```

Comment:
Isn't this just ValidEncoding with a fake moustache?

Reply:
heh, yeah, missed one in the search filter